next.config.js

```diff
@@ -13,7 +13,7 @@ const nextConfig = {
   experimental: {
     appDir: true,
   },
-  reactStrictMode: true,
+  reactStrictMode: false,
```

Comment:
This is not recommended 😅 Why are we changing it ?

Reply:
It is to prevent useEffect from running twice